util/dvsim/utils.py

```diff
@@ -209,14 +209,23 @@ def htmc_color_pc_cells(text):
     '''This function finds cells in a html table that contains a "%" sign. It then
     uses the number in front if the % sign to color the cell based on the value
     from a shade from red to green. These color styles are encoded in ./style.css
-    which is assumed to be accessible by the final webpage.
+    which is assumed to be accessible by the final webpage. If no markup shall be
+    applied to a percentage, the identifier "%u" can be used.
 
     This function is now augmented to also take "E" or "W" as identifiers along
     with "%". For example, '10 W' is indicative of 10 warnings, and will be color
     coded with yellow. Likewise, "7 E" indicates 7 errors and will be color coded
-    with red. A value of 0 in both cases will be color coded with green.
+    with red. A value <= 0 in both cases will be color coded with green.
+    Negative thresholded Error and Warning identifiers "EN" and "WN" are also
+    available, where values >= are color coded with green, and everything below
+    0 is color coded with red or yellow, respectively.
 
-    Note that a space between the value and the indicators (%, E, W) is mandatory.
+    The "G" and "B" identifiers can be used to implement custom highlighting
+    based on external logic, and refer to "good" and "bad", which turn into
+    "green" or "red" cells, respectively.
+
+    Note that a space between the value and the indicators
+    (%, E, W, EN, WN, G, B) is mandatory.
```

Comment:
Would you please rewrite this section in a more clear way:
``` 
  This function finds cells in a html table that contain numerical values (and a few known strings) followed by a single 
  space and an identifier. Depending on the identifier, it shades the cell in a specific way. A set of 12 color palettes for 
  setting those shades are encoded in ./style.css. These are 'cna' (grey), 'c0' (red), 'c1' ... 'c10' (green). The shade 'cna' 
  is used for items that are maked as 'not applicable'. The shades 'c1' to 'c9' form a gradient from red to lime-green to 
  indicate 'levels of completeness'. 'cna' is used for greying out a box for 'not  applicable' items, 'c0' for items that are 
  considered risky (or not yet started) and 'c10' for items that are completed successfully, or 'in good standing'.

  These are the supported identifiers: %, %u, G, B, E, W, EN, WN. The shading behavior for these is described below.    
  <identifier>:
  <Description>

  <identifier>:
  <Description>
```

Reply:
Yes that is better, thanks.